addons/udes_stock/wizard/change_quant_location.py

```diff
@@ -22,11 +21,57 @@ def _default_stock(self):
     )
     reference = fields.Char('Reference')
 
+    @api.multi
+    def check_reserved(self):
+        """Check if we are moving reserved pacakges and if that is permitted"""
+        self.ensure_one()
+        Package = self.env['stock.quant.package']
```

Comment:
(Feel free to ignore/resolve) Nit: I think `Package` should be placed above `self.ensure_one()` to follow convention. But looking around other methods it seems to be a mix over what should take precedent.